addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -199,6 +199,7 @@ var WysiwygMultizone = Wysiwyg.extend({
             self._getEditableArea().attr('contenteditable', true);
             self.$('[data-oe-readonly]').addClass('o_not_editable').attr('contenteditable', false);
             self.$('.oe_structure').attr('contenteditable', false).addClass('o_fake_not_editable');
+            self.$('.oe_structure').children().attr('contenteditable', true);
```

Comment:
Why was it working before ? And why not contenteditable on the oe_structure itself ?

Reply:
Let's see about that first question with @Gorash when he comes back because he's the one who wrote that code. He validated this fix though (while I did find it strange to treat the symptom instead of the cause, he found this the be the right thing to do in this particular case).
As for the second question, we want the snippets to be editable, not their container. Where you used to have an `iframe` per snippet you now have a `contenteditable` area per snippet. We certainly would want it to be possible to edit in between snippets.

Reply:
(there was not an iframe by snippet though, the editor was bad but not that bad ^^)

Reply:
(haha well I wasn't there, multizone was already done when I joined the project... the iframe thing is just what @Gorash explained to me :-D Now I'm curious though ^^)